Documentation/community/roadmap.rst

```diff
@@ -72,43 +97,28 @@ Cilium has applied for `CNCF Graduation`_, please add your support on the PR!
 Cilium Service Mesh 
 ~~~~~~~~~~~~~~~~~~~
 
-Our eBPF-accelerated Service Mesh is the main focus for
-major enhancement, and it's a natural evolution of Cilium's networking
-capabilities. We released a beta at the end of 2021 and had very valuable
-feedback from our user community. The next steps we'd like to take for Cilium
-Service Mesh (in no particular order) are: 
+As Cilium Service Mesh gets wider adoption, the next steps we'd like to take in
+this area (in no particular order) are: 
 
 * Graduating Prometheus metrics and OpenTelemetry collector to stable
-* Using Kubernetes as service mesh control plane 
- 
-  * Simple to use sidecar-free service mesh configured using Kubernetes Services
-    and Ingress with support for additional annotations
-
-* Graduating EnvoyConfig CRD to stable
 * Extended sample Grafana dashboards for L7 visibility
 * SMI integration 
-* SPIFFE integration
-* Gateway API Integration
```

Comment:
Probably worth retaining Gateway API in some form since the spec isn't finalized yet but we're working to implement newer versions of the spec?

Reply:
Agreed, I'd like to see Gateway API integration stay in next steps until it's GA.

Reply:
yep, added bullet for this